spring-cloud-alibaba-starters/spring-cloud-alibaba-commons/src/main/java/com/alibaba/cloud/commons/governance/auth/condition/AuthCondition.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013-2018 the original author or authors.
+ * Copyright 2022-2023 the original author or authors.
```

Comment:
The initial year should be kept, in this case 2013-2023

Reply:
Ah, I found that I was misunderstood. Your change is correct as this is a new module.